controllers/categories.js

```diff
@@ -0,0 +1,20 @@
+var express = require('express');
+var db = require('../models');
+var router = express.Router();
+var bodyParser = require('body-parser');
+var ejsLayouts = require('express-ejs-layouts');
+var app = express();
+
+router.get('/', function(req, res) {
+  db.category.findAll()
+  .then(function(categories) {
+  res.render('categories/index', { categories: categories });
+  })
+});
+
+
```

Comment:
you are close to finishing, just need to add something to the findAll, 

`db.category.findAll({
    where: {id: req.params.id}
 }).then(function(...`